fdbserver/TLogServer.actor.cpp

```diff
@@ -2378,8 +2384,13 @@ ACTOR Future<Void> tLogCommit(TLogData* self,
 		return Void();
 	}
 
+	// Measure server-side RPC latency from the time a request was
+	// received to time the response was sent.
+	const double endTime = g_network->timer();
+
 	if (isNotDuplicate) {
-		self->commitLatencyDist->sampleSeconds(now() - beforeCommitT);
+		self->commitLatencyDist->sampleSeconds(endTime - beforeCommitT);
```

Comment:
Presumably this is on purpose, but this changes the behavior from using the run loop clock to using the system clock. Is this change desired?

Reply:
Yes, this is on purpose. now() is only updated in the run loop. For accurate latency measurements, we need to use one of the real system clocks, either timer() or timer_monotonic().